Library/Formula/apache-spark.rb

```diff
@@ -1,13 +1,20 @@
 class ApacheSpark < Formula
   desc "Engine for large-scale data processing"
   homepage "https://spark.apache.org/"
-  url "https://www.apache.org/dyn/closer.lua?path=spark/spark-1.6.0/spark-1.6.0-bin-hadoop2.6.tgz"
   version "1.6.0"
-  sha256 "439fe7793e0725492d3d36448adcd1db38f438dd1392bffd556b58bb9a3a2601"
   head "https://github.com/apache/spark.git"
-
   bottle :unneeded
 
+  option "without-hadoop", "Use the -without-hadoop distribution of Spark (assumes you have a local installation of hadoop)"  
```

Comment:
I'm _pretty sure_ this is going to break our logic around the way we expect formulae to be structured, to be honest.